cli/lsp/config.rs

```diff
@@ -1663,7 +1669,21 @@ impl ConfigData {
         sloppy_imports_options: if unstable_sloppy_imports {
           SloppyImportsOptions::Enabled
         } else {
-          SloppyImportsOptions::Disabled
+          let compiler_options_resolver = compiler_options_resolver.clone();
+          SloppyImportsOptions::Dynamic(Arc::new(move |referrer| {
+            if let Some(referrer) = referrer {
+              let resolver = compiler_options_resolver.load();
```

Comment:
👍

I'll probably have to restructure the subsystem init sequence to clean this up.

Before:
1. `ConfigTree` (including `WorkspaceResolver`)
2. `LspResolver` (including `NodeResolver` and `CliResolver`)
3. `CompilerOptionsResolver`

After:
1. `ConfigTree`
2. `NodeResolver`
3. `CompilerOptionsResolver`
4. `WorkspaceResolver`
5. `CliResolver`